MANIFEST.ini

```diff
@@ -0,0 +1,3 @@
+include LICENSE
+include README.md
+recursive-include libzim_linux-x86_64-6.1.1 *
```

Comment:
This lets us optionally ship the libzim.so binary with the bindings it's compiled against it, so the user doesn't have to install it separately, and doesn't risk accidentally installing a different version system-wide than the ones the bindings were compiled against. (it will still use the system-installed dylib first, but the user can force it to use this inbuilt shipped one with some env vars if they don't want to install it separately)

Let me know if this is ok, or if you want us to remove this and force the user to always install libzim on the system separately.

Reply:
No it is good to include the libzim.so binary in the bdist.

However, I don't understand the `*` at the end of the line. We should include only the `.so` no ?

Reply:
The headers must be included in the `sdist` (because they are needed for Cython to be able to run), but they are not included in the `bdist` (which does not need the headers because it's already compiled into a self-contained .so).

Reply:
I would say that we don't want to include the library (and its header) in the sdist.
If the user want the source, it is to develop and he can compile the library itself (or dl a official release).
Including the `.so` in the bdist is useful to just `pip install` and use.

The common scheme is to not include dependencies in the sources (whatever the project language is).